### PR TITLE
Allow dynamic actions to be matched with `permit_action`

### PR DESCRIPTION
**Pull request by icnagy**
Fixing https://github.com/punditcommunity/pundit-matchers/issues/101

3.0.0 broke the permit_action matcher for policies with dynamic actions. Here's an attempt to bring back the pre-3.0.0 behaviour.

### CODE REVIEW

File: Dockerfile
Comment (tagliala) [inline Dockerfile:1]: Can you please make this change in a different PR?

**Review by tagliala — CHANGES_REQUESTED** [review]
Thanks, good catch

This would also require a line in the changelog and a bump of the patch version number

```
## 3.0.1 (unreleased)

- ...short, user friendly description of the fix...
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Pundit Matchers
 
+## 3.0.1 (unreleased)
+
+- Allow `permit_action` to match dynamic actions
```
Comment (tagliala) [inline CHANGELOG.md:5]: This will also allow `forbid_action`, `permit_actions`, and `forbid_actions`. @chrisalley do you have a better suggestion? Should `permit_action` just be omitted?

**Review by tagliala — APPROVED** [review]

**Comment by chrisalley** [comment]
> @tagliala anything I can do on my end to make the CI run pass?

Run `bundle` and commit `Gemfile.lock` which will contain the new gem version number.

File: CHANGELOG.md (hunk 1 above)
Comment (chrisalley) [reply CHANGELOG.md:5]: I would think that the four matchers should behave in a consistent way.

Comment (tagliala) [reply CHANGELOG.md:5]: Sorry, I meant that this fix applies to all the matchers not just `permit_actions`, so this can be rephrased as

```
- Fix a regression that prevented to match dynamically defined actions
```

```
- Allow to match dynamically defined actions

```
- Allow to match dynamic actions
```

or something like this

**Comment by icnagy** [comment]
Sorry folks, it's impossible to resolve the last conversation because I force pushed my branch and the relevant commit is now out of reach.

**Review by tagliala — CHANGES_REQUESTED** [review]

**Review by tagliala — APPROVED** [review]

**Comment by icnagy** [comment]
Thank you @tagliala @chrisalley for the guidance/help!

**Review by chrisalley — APPROVED** [review]
